### PR TITLE
bug fixed

### PR DESCRIPTION
**Pull request by Eloahman**
# self.protocol.callPing(bucket.head())
 # bug fixed: Use synchronous call to an asynchronous function
 asyncio.ensure_future(self.protocol.callPing(bucket.head()))

I just started to use github, so don't be puzzled to see so many commits. : )
And would you mind I ask a question? I'm a bit confused in the using asyncio. When I create a loop in class server, should I create another loop in the class ForgetfulStorage if I want to do something asynchronous in the storage? So where exactly the loop in? In the instance? It seems to be everywhere in my code. : )

### CODE REVIEW

File: kademlia/routing.py [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import heapq
 import time
 import operator
-from collections import OrderedDict
+import asyncio
 
+from collections import OrderedDict
```
Comment (bmuller) [inline kademlia/routing.py]: Why was this one moved down?

**Comment by bmuller** [comment]
Good catch!  I had one comment just wondering why a line was moved - other than that thanks for catching this!

There should only be one loop - and `asyncio.get_event_loop()` will always return that one instance.

File: kademlia/routing.py (hunk 1 above)
Comment (Eloahman) [inline kademlia/routing.py]: Ooops. Nothing serious. That's a little trick that I don't know why git can't discover my change so I have to make another "change" to remind it. If you think it's not good, I would try to fix it. : )

Comment (bmuller) [inline kademlia/routing.py]: Unnecessary changes should not be committed as part of a pull request.  If you move this line back, then I'll happily merge.  Thanks!

Comment (Eloahman) [inline kademlia/routing.py]: I'm so sorry for that.

**Pull request merged**